airbyte-server/src/main/java/io/airbyte/server/handlers/WorkspacesHandler.java

```diff
@@ -141,6 +144,13 @@ public void deleteWorkspace(final WorkspaceIdRequestBody workspaceIdRequestBody)
     configRepository.writeStandardWorkspace(persistedWorkspace);
   }
 
+  public WorkspaceReadList listWorkspaces() throws JsonValidationException, IOException {
```

Comment:
Could you add unit test to verify this? You can add in `WorkspacesHandlerTest.java`

Reply:
Sure! Just add it @sherifnada ;-)